core/src/test/java/fr/sncf/osrd/api/InfraLoadingTest.java

```diff
@@ -28,6 +29,20 @@ public void simple() throws Exception {
                 "tiny_infra/infra.json",
                 "1"
         );
-        assertEquals("Infra loaded", runInfraLoad(query));
+
+        assert contains200(runInfraLoad(query));
+    }
+
+    /** Returns true if the response is a 200 */
+    private static boolean contains200(List<String> res) {
```

Comment:
If you'd like, iIcan show you into the world of Streams... Here's how your code could be simplified with the Java Stream object :)
```
return res.stream()
   .anyMatch(header -> header.contains("200"));
```
By the way, this looks like a method that could be useful in a lot of other situations. You could put it in an Utils class and name it `assertListContainsString(List<String> words, String word)` (or w/e other matching name you come up with).
Another good practice is to put "200" into a private static final constant in the class (normally, tools like sonar would say that "200" is a magic number, since they don't know what it is, and it would be considered a code smell).